sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -937,6 +954,23 @@ object Radar {
         return RadarSettings.getTrackingOptions(context)
     }
 
+    /**
+     * Settings for the foreground notification when the foregroundServiceEnabled parameter
+     * is true on Radar tracking options.
+     *
+     * @see [](https://radar.io/documentation/sdk/tracking)
```

Comment:
Missing link text?

Reply:
@jrtibbetts these are scattered all across the sdk. I believe this is a _shortcut_ for dokka, and not necessarily a markdown bug, but I have not looked into it thoroughly.

Reply:
yeah, this is the format used across the codebase. 

I suspect this interpretation of markdown uses the URL as the text as well.

however, we should have a task to write docs for remote tracking options, and link to that cc @jsani-radar

Reply:
TIL. Thanks.

Reply:
I added a task on Shortcut.